src/main/java/org/folio/bulkops/service/UserPermissionsService.java

```diff
@@ -54,8 +55,8 @@ public List<String> getPermissions() {
   private List<String> getDesiredPermissions() {
     return List.of(BULK_EDIT_INVENTORY_WRITE_PERMISSION.getValue(), BULK_EDIT_USERS_WRITE_PERMISSION.getValue(),
       USERS_ITEM_PUT.getValue(), INVENTORY_ITEMS_ITEM_PUT.getValue(), INVENTORY_STORAGE_HOLDINGS_ITEM_PUT.getValue(),
-            INVENTORY_INSTANCES_ITEM_PUT.getValue(), BULK_EDIT_INVENTORY_VIEW_PERMISSION.getValue(), BULK_EDIT_USERS_VIEW_PERMISSION.getValue(),
-            USER_ITEM_GET_PERMISSION.getValue(), INVENTORY_ITEMS_ITEM_GET_PERMISSION.getValue(),
-            INVENTORY_INSTANCES_ITEM_GET_PERMISSION.getValue(), INVENTORY_STORAGE_HOLDINGS_ITEM_GET_PERMISSION.getValue());
+      INVENTORY_INSTANCES_ITEM_PUT.getValue(), BULK_EDIT_INVENTORY_VIEW_PERMISSION.getValue(), BULK_EDIT_USERS_VIEW_PERMISSION.getValue(),
+      USER_ITEM_GET_PERMISSION.getValue(), INVENTORY_ITEMS_ITEM_GET_PERMISSION.getValue(),
+      INVENTORY_INSTANCES_ITEM_GET_PERMISSION.getValue(), INVENTORY_STORAGE_HOLDINGS_ITEM_GET_PERMISSION.getValue(), BULK_OPERATIONS_PROFILES_ITEM_LOCK.getValue());
```

Comment:
looks at https://github.com/folio-org/mod-invoice/blob/master/descriptors/ModuleDescriptor-template.json#L60 , it's an example how desired perms work on module level, `getDesiredPermissions` is util to work with a case when permissions are declared in other modules